### ISSUE TITLE
Need to use REST from different domain/port 

### ISSUE THREAD

**Issue by raitisbe** [jirik/layman#18]
CORS headers might solve it. I tried running with proxy http://localhost/layman -> http://localhost:8000 , but got 

layman_dev           |   File "/usr/local/lib/python3.6/dist-packages/werkzeug/datastructures.py", line 442, in __getitem__
layman_dev           |     raise exceptions.BadRequestKeyError(key)
layman_dev           | werkzeug.exceptions.HTTPException.wrap.<locals>.newcls: 400 Bad Request: The browser (or proxy) sent a request that this server could not understand.

**Comment by raitisbe** [comment]
Never mind the BadRequestKeyError(key) error. It was probably due to my lack of knowledge about nginx proxy. Finally got it working with proxy_pass and some rewritten headers, but still setting up proxy is a bit cumbersome and CORS headers would be the preffered method in my opinion.

**Comment by jirik** [comment]
Not sure what is your use case, but if you are using nginx, you can setup CORS headers just with nginx:
- https://enable-cors.org/server_nginx.html
- https://serverfault.com/questions/674900/nginx-if-statement-inside-location-returns-404/674901#674901
- https://www.nginx.com/resources/wiki/start/topics/depth/ifisevil/

Anyway, thanks for the suggestion. Sounds to me like a good idea to enable CORS headers directly in Layman. It can be controlled by environment variables.

**Comment by jirik** [comment]
Few months later, I feel that settings CORS headers on nginx is sufficient and that we do not need to implement it on Layman.

@raitisbe What's your opinion currently?
